rust/src/lib.rs

```diff
@@ -82,6 +82,34 @@ mod ffi {
         fn progress_end(suffix: &str);
     }
 
+    // history.rs
+
+    /// A history entry in the journal. It may represent multiple consecutive boots
+    /// into the same deployment. This struct is exposed directly via FFI to C.
+    #[derive(PartialEq, Debug)]
+    pub struct HistoryEntry {
```

Comment:
Can we keep this in `history.rs` and just wrap it around another `#[cxx::bridge(namespace = "rpmostreecxx")]`? Or will it not allow multiple bridges for the same namespace?

Reply:
cxx-rs only sees things in the macro.  Longer term, we can probably go to multiple bridge macros, there's some support for intermixing them (discussed a bit in https://cxx.rs/extern-c++.html ) but I haven't yet wanted to dig into that level.

See also https://github.com/dtolnay/cxx/issues/670